subprojects/build-scan-performance/src/performanceTest/groovy/org/gradle/performance/BuildScanPluginPerformanceTest.groovy

```diff
@@ -37,13 +41,20 @@ class BuildScanPluginPerformanceTest extends AbstractBuildScanPluginPerformanceT
     @Unroll
     def "with and without plugin application (#scenario)"() {
         given:
+        def baselineVersions = resolveBaselineVersions(System.getProperty('org.gradle.performance.baselines'), ['last'])
```

Comment:
Can't you use `toBaselineVersions()` here as well?

Reply:
fixed